src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -56,7 +67,15 @@ public bool Display
         set
         {
             display = value;
-            UpdateToolTipVisibility();
+
+            if (previousToolTip != null)
+                FinalizeToolTipVisibility(previousToolTip, false);
+
+            if (display)
+            {
+                // Set timer
+                displayTimer = MainToolTip.DisplayDelay;
+            }
```

Comment:
is it beneficial to run this code when display was already true?

Reply:
Yes, consider one of a likely scenario: `MainToolTip` was reassigned to another tooltip from an initial tooltip with `Display` already set true, and assuming the newly assigned tooltip is hidden we could make it visible by setting `Display` to true once more (basically to reset the display timer).